java/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java-library'
+}
+
+group = 'com.mongodb.csfle'
+
+version = '0.1-SNAPSHOT'
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(
+        [
+            'org.mongodb:mongodb-crypt:1.0.0',
+            'org.mongodb:mongodb-driver-sync:3.11.2'
```

Comment:
This will probably end up being 3.12.0

Reply:
Thanks, we'll update this once the new version is released.

Reply:
Resolving this, will take action when new driver released.